docs/usage/administration/admin_faq.md

```diff
@@ -32,9 +32,25 @@ What users are registered on my server?
 SELECT NAME from users;
 ```
 
-Manually resetting passwords:
+Manually resetting passwords
 ---
-See https://github.com/matrix-org/synapse/blob/master/README.rst#password-reset
+Users can reset their password through their client. Alternatively, a server admin
+can reset a user's password using the [admin API](../../admin_api/user_admin_api.md#reset-password)
+or by directly editing the database as shown below.
```

Comment:
maybe add a caveat that manually editing the DB is generally not recommended?

Reply:
I'm more inclined to remove the manual instructions, tbh.

Reply:
2ce408eb3